cst/plugins/eu.esdihumboldt.cst.functions.geometric.test/src/eu/esdihumboldt/cst/functions/geometric/TransformationReproject.java

```diff
@@ -0,0 +1,227 @@
+package eu.esdihumboldt.cst.functions.geometric;
+
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import eu.esdihumboldt.cst.test.TransformationExample;
+import eu.esdihumboldt.hale.common.align.model.Alignment;
+import eu.esdihumboldt.hale.common.core.io.supplier.LocatableInputSupplier;
+import eu.esdihumboldt.hale.common.instance.model.Filter;
+import eu.esdihumboldt.hale.common.instance.model.Instance;
+import eu.esdihumboldt.hale.common.instance.model.InstanceCollection;
+import eu.esdihumboldt.hale.common.instance.model.InstanceReference;
+import eu.esdihumboldt.hale.common.instance.model.ResourceIterator;
+import eu.esdihumboldt.hale.common.instance.model.ext.InstanceIterator;
+import eu.esdihumboldt.hale.common.instance.model.impl.FilteredInstanceCollection;
+import eu.esdihumboldt.hale.common.instance.model.impl.PseudoInstanceReference;
+import eu.esdihumboldt.hale.common.schema.model.Schema;
+import eu.esdihumboldt.hale.common.schema.model.TypeDefinition;
+import eu.esdihumboldt.hale.common.test.TestUtil;
+
+public class TransformationReproject implements TransformationExample{
```

Comment:
As far as I see the functionality in here is more or less the same as in the existing `TransformationExample` implementation `InternalExample`. As it is not reusable like that in another plugin, I extracted a base class (`TransformationExampleImpl`) from it, that you should be able to use and thus get rid of most of this file's content, so we avoid any duplication.

The class mentioned was added recently in 4a349f770727e11097fbc867572d316cda8b15b0.